src/cmd/create.go

```diff
@@ -699,9 +700,14 @@ func pullImage(image, release, authFile string) (bool, error) {
 	}
 
 	if promptForDownload {
+		infoSize, err := skopeo.Inspect(imageFull)
+		if err != nil {
+			logrus.Debug("cannot get the size of the image")
```

Comment:
Since we are suppressing the original error, it will be good to dump it in the logs.  Something like:
```
  logrus.Debugf("Getting image size failed: %s", err)
```